.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+---
+version: 2.1
+
+executors:
+
+jobs:
+  test:
+    parameters:
+      goarch:
+        type: string
+      goversion:
+        type: string
+    docker:
+      - image: circleci/golang:<< parameters.goversion >>
+    environment:
+      GOARCH: << parameters.goarch >>
+      GOSNMP_TARGET: "127.0.0.1"
+      GOSNMP_PORT: "161"
+      GOSNMP_TARGET_IPV4: "127.0.0.1"
+      GOSNMP_PORT_IPV4: "161"
+      GOSNMP_TARGET_IPV6: "::1"
+      GOSNMP_PORT_IPV6: "161"
+
+    steps:
+    - checkout
+    - run: sudo apt-get update
+    - run: sudo apt-get -y install snmpd
+    - run: sudo ./snmp_users.sh
+    - run: sudo /etc/init.d/snmpd restart
+    - run: make tools
+    - run: make lint
+    - run: go test -v -tags helper
+    - run: go test -v -tags marshal
+    - run: go test -v -tags misc
+    - run: go test -v -tags api
+    - run: go test -v -tags end2end
+    - run: go test -v -tags trap
+
```

Comment:
Could you also add `` -run: go test -v -tags all -race``

Reply:
```
go test: -race is only supported on linux/amd64, linux/ppc64le, linux/arm64, freebsd/amd64, netbsd/amd64, darwin/amd64 and windows/amd64
```

Whups, doesn't worth with the 386 builds.